tests/python/test_matrix.py

```diff
@@ -18,6 +18,23 @@ def test_python_scope_vector_operations():
         assert np.allclose(c.to_numpy(), ops(a, b))
 
 
+@ti.host_arch_only
+def test_taichi_scope_vector_operations_with_global_vectors():
+    for ops in vector_operation_types:
+        ti.reset()
```

Comment:
```suggestion
@pytest.mark.parametrize('ops', vector_operation_types)
@ti.host_arch_only
def test_taichi_scope_vector_operations_with_global_vectors(ops):
```
`pytest` is cool, isn't it?